makelib/variables.mk

```diff
@@ -16,12 +16,16 @@ FILELISTDIR := $(BUILDDIR)/filelists
 MAINTEMPDIR := $(BUILDDIR)/tmp
 CLEANDIR := $(BUILDDIR)/clean
 
+# host tools
 ACTOOL := $(TOOLSDIR)/actool
 DEPSGENTOOL := $(TOOLSDIR)/depsgen
 FILELISTGENTOOL := $(TOOLSDIR)/filelistgen
 CLEANGENTOOL := $(TOOLSDIR)/cleangen
 QUICKRMTOOL := $(TOOLSDIR)/quickrm
 
+# target tools
+ACTOOL_TARGET := $(TARGET_BINDIR)/actool
```

Comment:
OK.  CoreOS currently builds and installs actool for the target along with rkt, and is why I added this.  I'll look into adding a new CoreOS ebuild for actool.